src/views/bibliography_element.ts

```diff
@@ -123,10 +119,17 @@ export class BibliographyElementBlockView<
   }
 
   private handleClick = (event: Event) => {
-    this.props.popper.destroy()
     const element = event.target as HTMLElement
     const item = element.closest('.bib-item')
     if (item) {
+      const marker = element.closest('.comment-marker') as HTMLElement
+      if (marker) {
+        const key = marker.dataset.key as CommentKey
+        const tr = this.view.state.tr
+        setCommentSelection(tr, key, undefined, false)
+        this.view.dispatch(tr)
+        return
+      }
```

Comment:
we need to do that also to keyword

Reply:
We should not need that, the plugin should handle it.

Reply:
oh it's working, now it was about `stopEvent`